### PR TITLE
git-imerge 1.2.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
license header ref: 
```
# -*- coding: utf-8 -*-

# Copyright 2012-2013 Michael Haggerty <mhagger@alum.mit.edu>
#
# This file is part of git-imerge.
#
# git-imerge is free software: you can redistribute it and/or
# modify it under the terms of the GNU General Public License as
# published by the Free Software Foundation, either version 2 of the
# License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
# General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see
# <http://www.gnu.org/licenses/>.

```

File: Formula/git-imerge.rb [hunk 1]
```diff
@@ -15,13 +17,71 @@ class GitImerge < Formula
...
+    bash_completion.install "completions/git-imerge"
   end
 
   test do
-    system bin/"git-imerge", "-h"
+    system "git", "init"
+    system "git", "config", "user.name", "BrewTestBot"
+    system "git", "config", "user.email", "BrewTestBot@example.com"
+
+    system "echo \"puts 'Hello, world'\" > main.rb && git add main.rb && git commit -m \"First commit\""
```
Comment (dawidd6) [inline Formula/git-imerge.rb:36]: Could this be split into separate system calls? And also use `(testpath/"file").write`?

Comment (chenrui333) [reply Formula/git-imerge.rb:36]: 👍

**Comment by chenrui333** [comment]
Close in favor of #61745

**Pull request closed without merging**